constructor/nsis/main.nsi.tmpl

```diff
@@ -881,7 +881,7 @@ Section "Install"
 
     SetDetailsPrint TextOnly
     DetailPrint "Setting up the base environment ..."
-    nsExec::ExecToLog '"$INSTDIR\_conda.exe" install --offline -yp "$INSTDIR" --file "$INSTDIR\pkgs\env.txt"'
+    nsExec::ExecToLog '"$INSTDIR\_conda.exe" install --offline -yp "$INSTDIR" --file "$INSTDIR\pkgs\env.txt" --no-shortcuts'
```

Comment:
What does this parameter do?

Reply:
This doesn't create the start menu shortcuts. We don't want to create the shortcuts with `conda install` because we do it in the following step with a dedicated option of conda-standalone in order to support the `menu_packages` option of `constructor`:
```
"$INSTDIR\_conda.exe" constructor --prefix "$INSTDIR" --make-menus @MENU_PKGS@
```
I added a comment explaining it.

Reply:
Ah, got it, thank you!